src/main/java/me/jellysquid/mods/phosphor/mixin/chunk/light/MixinChunkLightProvider.java

```diff
@@ -60,10 +60,8 @@ protected MixinChunkLightProvider(int levelCount, int expectedLevelSize, int exp
     @Inject(method = "clearChunkCache", at = @At("RETURN"))
     private void onCleanup(CallbackInfo ci) {
         // This callback may be executed from the constructor above, and the object won't be initialized then
-        if (this.cachedChunkPos != null) {
-            Arrays.fill(this.cachedChunkPos, Long.MIN_VALUE);
-            Arrays.fill(this.cachedChunkSections, null);
-        }
+        Arrays.fill(this.cachedChunkPos, Long.MIN_VALUE);
```

Comment:
The null-check has been removed, but the comment remains without any validation here that it's no longer necessary. While most automated tools will think this is unnecessary, but these variables are initialized in `<init>` which is always called after the super class runs its own constructor. Has this change been tested?

Reply:
Oh I missed the comment. I will remove it. I have tested the whole place and didn't get any NPEs or similar.